tests/files/test_sbatch.py

```diff
@@ -10,27 +10,33 @@ def test_SbatchScript(tmpdir):
     tmp_dir = Path(tmpdir.mkdir('test_write_input'))
 
 
-    sbatch_script = SbatchScript('mutacc_env','stdout_file','stderr_file','email',
-                                 tmp_dir)
+    with SbatchScript(tmp_dir,
+                      'mutacc_env',
+                      tmp_dir,
+                      'email') as sbatch_script:
 
-    sbatch_script.write_section('Command to run')
+        sbatch_script.write_section('Command to run')
 
     sbatch_path = sbatch_script.path
 
     assert Path(sbatch_path).exists()
 
-    sbatch_script = SbatchScript('mutacc_env','stdout_file','stderr_file','email',
-                                 tmp_dir, conda=True)
+    with SbatchScript(tmp_dir,
+                     'mutacc_env',
+                      tmp_dir) as sbatch_script:
 
-    sbatch_script.write_section('Command to run')
+        sbatch_script.write_section('Command to run')
 
     sbatch_path = sbatch_script.path
 
     assert Path(sbatch_path).exists()
 
+    with pytest.raises(FileNotFoundError) as error:
+
+        sbatch_script = SbatchScript(tmp_dir,
+                                     'mutacc_env',
+                                     '/fds9Xs/fdsuerw')
```

Comment:
'/fds9Xs/fdsuerw'
This looks very hard coded! What is it? A temp dir again?

Reply:
This is just to test the function with a nonsense path. will change.